Decred Wallet/Utils/DecredAddressURI.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DecredAddressURI.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+
+struct DecredAddressURI {
+    var amount: String?
+    var address: String?
+    var memo: String?
```

Comment:
This is not what I have meant eighter. You should not search the `memo` variable because it was just an example. My point was that there could be more than one query

Reply:
resolved in `1e660b7`